### PR TITLE
Fixed Dockerfile to make use of dep

### PR DESCRIPTION
**Pull request by haxorof**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,9 +1,11 @@
 # Requires Docker v17.06 or later
-FROM golang:1.8.3 as builder
+FROM golang:1.8.5 as builder
```
Comment (bruth) [inline Dockerfile]: Any reason not to jump up to 1.9.2?

File: Gopkg.lock [hunk 2]
```diff
@@ -7,11 +7,17 @@
   packages = ["quantile"]
   revision = "4c0e84591b9aa9e6dcfdf3e020114cd81f89d5f9"
 
+[[projects]]
+  name = "github.com/gogo/protobuf"
+  packages = ["proto"]
+  revision = "342cbe0a04158f6dcb03ca0079991a51a4248c02"
+  version = "v0.5"
+
```
Comment (bruth) [inline Gopkg.lock]: Is prometheus using both `gogo/protobuf` and `golang/protobuf` (below)?

File: Dockerfile (hunk 1 above)
Comment (haxorof) [reply Dockerfile]: I was just cautious. 😄 Shall I step it up to 1.9.2 then? Should I also increase to Alpine 3.6 at the same time?

File: Gopkg.lock (hunk 2 above)
Comment (haxorof) [reply Gopkg.lock]: I am not so familiar to dep but this was added then I executed `dep ensure` and this file shall not be modified manually.

File: Dockerfile (hunk 1 above)
Comment (bruth) [reply Dockerfile]: Ha, yea I think it is safe. 👍 for Alpine.

File: Gopkg.lock (hunk 2 above)
Comment (bruth) [reply Gopkg.lock]: Right. Ok, it may be from two different deps.

File: .gitignore [hunk 3]
```diff
@@ -4,6 +4,7 @@ prometheus-sql.exe
 .idea
 .vagrant
 vendor
+vendor.orig
```
Comment (bruth) [inline .gitignore]: Any reason for adding this? Seems like a temporary inclusion based on this transition for your local repo.

**Comment by haxorof** [comment]
Now I increase both Golang and Alpine version. 😄

**Pull request merged**